sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -132,7 +147,55 @@ export class SupportingUserController {
       >
     >,
   ): Promise<MustReturnJobActionAcknowledgement> {
-    return job.fail("To be implemented.");
+    const jobLogger = new Logger(job.type);
+    try {
+      const application = await this.applicationService.getApplicationById(
+        job.variables.applicationId,
+        { loadDynamicData: true },
+      );
+      if (!application) {
+        const message = "Application ID not found.";
+        jobLogger.error(message);
+        return job.error(APPLICATION_NOT_FOUND, message);
+      }
+      // Resolve the full name from the application dynamic data using the optional provided filter.
+      let fullName: string;
+      if (job.variables.fullNamePropertyFilter) {
+        [fullName] = JSONPath.query(
+          application.data,
+          job.variables.fullNamePropertyFilter,
+        );
+        if (!fullName) {
+          // If the full name property filter is provided, full name is mandatory to be present.
+          const message = `Not able to extract the full name from the application dynamic data using filter '${job.variables.fullNamePropertyFilter}'.`;
+          jobLogger.error(message);
+          return job.error(SUPPORTING_USER_FULL_NAME_NOT_RESOLVED, message);
+        }
+      }
+      return this.dataSource.transaction(
+        // Transaction created with the expectation that a notification will also be created
+        // for the supporting user and it should be saved in the same transaction.
+        async (entityManager: EntityManager) => {
+          const createdSupportingUserId =
+            await this.supportingUserService.createIdentifiableSupportingUser(
+              {
+                applicationId: job.variables.applicationId,
+                supportingUserType: job.variables.supportingUserType,
+                fullName,
+                isAbleToReport: job.variables.isAbleToReport,
```

Comment:
Looking at the feature broadly now, is `isAbleToReport` required in the context of workflow, when it can be retrieved from the application data.? Let me know if I am missing somthing.